### DISCUSSION TITLE
请问如何代理数据库连接？

### DISCUSSION THREAD

**Discussion by OneEyedOlaf** [v2fly/v2ray-core#3184] [Q&A]
我家里的电信网络连不上数据库，但是移动的网络就可以连，我排查一下后似乎数据库连接没有走代理，该如何解决呢？

**Comment by tomandjerry27** [comment]
不是所有软件都默认使用系统代理设置的呀，我看navicat里边就有一个选项，你可以自己选要不要使用系统代理设置。
第二个选择就是把代理放到网关上做无感代理，这样就不需要在电脑上配置了。
第三个方案呢就是电脑两个网卡一个连电信网，一个连移动网。然后往系统里加一条静态路由，让去数据库的目的ip能匹配到移动网的出接口。
其实还有可能是你的代理ip条目里你数据库的地址是直连的，即便流量到了代理软件也不会转发到代理服务器。这样的话你就需要修改config.json配置。
你自己排查一下你是什么问题。
